### ISSUE TITLE
Re-joining a private message channel does not populate history

### ISSUE THREAD

**Issue by peppy** [ppy/osu#9113]

**Comment by EZ2ACTux** [comment]
Interestingly if you rejoin through the browser and then go to the client your message history does populate.

**Comment by peppy** [comment]
Yep, it's specific to the new channel creation flow. Should be pretty easy to see why (I don't think the call is run to populate it in this scenario).

**Comment by EZ2ACTux** [comment]
Would we be able to just run the call to populate for every DM instance or will that cause conflicts if you try to open a DM with someone who you haven't opened a chat with before?

**Comment by peppy** [comment]
Should be fine.

**Comment by EZ2ACTux** [comment]
Will see what I can do

**Comment by EZ2ACTux** [comment]
Ok this is strange, after setting a breakpoint on the command to fetch the backlog I can confirm it is indeed being tripped after reopening a PM channel but the messages aren't downloading (see attached images)

![image](https://user-images.githubusercontent.com/10949323/82878506-aecc8480-9f8f-11ea-9dae-20b8ecd14831.png)

![image](https://user-images.githubusercontent.com/10949323/82878530-b855ec80-9f8f-11ea-81ac-d726953c7c15.png)

**Comment by EZ2ACTux** [comment]
Ok so what I think is happening is that lazer is simply trying to create a new channel instead of reconnecting to the existing one since even after manually stepping into the code to pull chat history I still didn't get anything.

I tried the technique I used to get my chats to show back up in lazer before the fix was deployed (So go to the osu website, open my friends list, start a chat from there then alt tab back into lazer) after having already opened a chat through lazer itself and I predicted, lazer had spawned two PM channels with the same person, one WITH chat history and one without.

Lazer is probably trying to create a new channel with a user instead of loading the existing one which would explain why there's no chat history, it's a new channel.

Chat Channel opened via Lazer
![image](https://user-images.githubusercontent.com/10949323/82880218-374c2480-9f92-11ea-9890-bbb5e0a6f079.png)

Chat Channel opened via osu!Web
![image](https://user-images.githubusercontent.com/10949323/82880252-416e2300-9f92-11ea-8177-a8de5a3ef84c.png)
